cmd/kube-batch/app/server.go

```diff
@@ -72,7 +71,7 @@ func buildConfig(opt *options.ServerOption) (*rest.Config, error) {
 	return cfg, nil
 }
 
-// Run the kubeBatch scheduler
+// Run the kube-batch scheduler
```

Comment:
Should be volcano scheduler.

Reply:
sorry, i missed this.

As there are many cleanup work left, let me open an issue